meson.build

```diff
@@ -288,6 +288,11 @@ test_env.set('top_builddir', meson.build_root())
 test_env.set('MALLOC_PERTURB_', '15')
 test_env.set('MallocPreScribble', '1')
 test_env.set('MallocScribble', '1')
+
```

Comment:
I introduced a conflict here in master (sorry) - needs a rebase.

Reply:
easy fix